### PR TITLE
[12.0][MIG] crm_sale_marketing

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: crm_sale_marketing/views/crm_sale_marketing.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="crm_case_form_view_oppor_extended">
```
Comment (tbaden) [inline crm_sale_marketing/views/crm_sale_marketing.xml:4]: optional:
id before model
and 
use original id(crm_case_form_view_oppor)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
If you like some optional changes

**Review by HviorForgeFlow — APPROVED** [review]
:+1:

**Review by rafaelbn — APPROVED** [review]

**Pull request closed without merging**